### PR TITLE
SC-9435 - Add Signl4 notification hook to docs

### PR DESCRIPTION
**Pull request by gr0**
Hi guys, 

This PR adds Signl4 notification hook integration to docs and looks like this:

signl4

### CODE REVIEW

**Review by adnanrahic — APPROVED** [review]

**Comment by adnanrahic** [comment]
This is good @gr0 , but please also add this notification hook to the docs/integration/overview.md in the `notification hooks` section.

**Comment by gr0** [comment]
Good point, I again forgot about it. Will do.

**Comment by gr0** [comment]
Actually, it is already there, I added it this time ;)

**Comment by otisg** [comment]
In general - for this hook or others, I think we can skip the "create account" step.  If one is looking at this documentation then one already has an account with the other service.

File: docs/integration/alerts-signl4-integration.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+title: Signl4 Alerts Integration
+description: Sematext threshold, anomaly and / or heartbeat Alerts integration with Signl4. Use them together to get the information about each notification quickly and on any device.
+
+## In Signl4
+
+**1.** Create a Signl4 account at [signl4.com](https://www.signl4.com).
```
Comment (otisg) [inline docs/integration/alerts-signl4-integration.md:6]: I'd zap this here and in all other hook pages

**Comment by adnanrahic** [comment]
Awesome! Good to merge then.

**Review by otisg — APPROVED** [review]

File: docs/integration/alerts-signl4-integration.md (hunk 1 above)
Comment (gr0) [reply docs/integration/alerts-signl4-integration.md:6]: Good point, I'll do that.

**Comment by gr0** [comment]
Thanks guys!

**Pull request merged**